release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -68,6 +68,10 @@ This release adds improvements related to the following components and concepts.
 In {product-title} {product-version}, you can disable the cloud controller manager capability during installation.
 For more information, see xref:../installing/cluster-capabilities.adoc#cluster-cloud-controller-manager-operator_cluster-capabilities[Cloud controller manager capability].
 
+[id="ocp-4-16-installation-and-update-kubeadmin_{context}"]
+==== The kubeadmin password no longer displayed in the console
+With this release, the {product-title} installation program no longer exposes the `kubeadmin` password in the console post installation owing to security reasons.The password continues to be available in the `auth` directory for installation.
```

Comment:
The feature must be enabled explicitly, by providing the parameter `--skip-password-print` to `openshift-install create cluster...`. Otherwise, the kubeadmin's password will be printed in the console, just as before. Thanks!

Reply:
oh I see! So this is not getting hidden by default. I will change the note.